### ISSUE TITLE
error compiling executable program with windows 64-bit with the latest code

### ISSUE THREAD

**Issue by YaakovHuang** [ergoplatform/cuda-miner#52]
When compiling the executable program miner.exe with the latest code in windows 64-bit, the error is over 100, then the compilation is stopped, and only test.exe is successfully compiled.

**Comment by c0mm4nd** [comment]
I have met the  same question. I tested that comiling the miner only (without test) and then I found that VS2017 **cannot compiling the japanese_str.h in BIP39 folder**.

After I deleting japanese chinese and other language, **leaving English only**(of course have to edit bip39.cc either), I **can** build the miner.

My env is Win10 CUDA 10.1 VS2017 GTX1063.

cuda10.1 seems that cannot run this miner correctly. I have no hashrate though receiving the task from node

**Comment by YaakovHuang** [comment]
> no hashrate though receiving the task from node

Yes, I also encountered this problem that no hashrate though receiving the task from node. now that I have stopped mining, I have found that it is too difficult to successfully mine according to the current calculations.
me env is Win10 CUDA 10.1 VS2017 GTX1070TI.

**Comment by YaakovHuang** [comment]
> I have met the same question. I tested that comiling the miner only (without test) and then I found that VS2017 **cannot compiling the japanese_str.h in BIP39 folder**.
> 
> After I deleting japanese chinese and other language, **leaving English only**(of course have to edit bip39.cc either), I **can** build the miner.
> 
> My env is Win10 CUDA 10.1 VS2017 GTX1063.
> 
> cuda10.1 seems that cannot run this miner correctly. I have no hashrate though receiving the task from node

Yes, I also encountered this problem that no hashrate though receiving the task from node. now that I have stopped mining, I have found that it is too difficult to successfully mine according to the current calculations.
me env is Win10 CUDA 10.1 VS2017 GTX1070TI.
